ether/cell.py

```diff
@@ -5,6 +5,7 @@
 
 from srds import RandomSampler, ConstantSampler, IntegerTruncationSampler
 
+from ether import distributions
```

Comment:
could we maybe name this module `qos` and have a submodule `latency` ?

so that it reads `from ether.qos import latency`